hlint.cabal

```diff
@@ -60,7 +60,9 @@ library
         extra >= 1.6.6,
         refact >= 0.3,
         aeson >= 1.1.2.0,
-        ghc-lib-parser == 8.8.1
+        ghc-lib-parser == 8.8.1,
+        syb >= 0.7.1,
+        mtl >= 2.2.2
```

Comment:
Usually transformers is a better choice.

Reply:
But not at the cost of rewriting copied over code!

Reply:
✔️ (won't fix).